packages/installation-tests/installation-tests.sh

```diff
@@ -194,6 +196,32 @@ function test_skip_browser_download {
   echo "${FUNCNAME[0]} success"
 }
 
+function test_skip_browser_download_inspect_with_custom_executable {
+  initialize_test "${FUNCNAME[0]}"
+  copy_test_scripts
+
+  OUTPUT=$(PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install ${PLAYWRIGHT_TGZ})
+  if [[ "${OUTPUT}" != *"Skipping browsers download because"* ]]; then
+    echo "missing log message that browsers download is skipped"
+    exit 1
+  fi
+
+  if [[ "$(uname)" != "Linux" ]]; then
+    echo
+    echo "Skipping test on non-Linux platform"
+    echo
+    return
+  fi
+
+  OUTPUT=$(PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 PWDEBUG=1 node inspector-custom-executable.js)
```

Comment:
Just do double-check: `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1` is not required here, right?

Reply:
Yes, you are right

Reply:
Removed.